.github/workflows/gradle.yml

```
@@ -0,0 +1,38 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will build a Java project with Gradle and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-gradle
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up JDK 11
+      uses: actions/setup-java@v3
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+
+    - name: Create application.properties
+      run: echo "${{ secrets.APPLICATION }}" > ./src/main/resources/application.yml
```

directory 생성하지 않아도 바로 yaml파일 생성 가능한지 궁금하네요!

---

```suggestion
      run: |
        cd ./src/main/resources
        echo "${{ secrets.APPLICATION }}" | base64 -d > application.yml
```
뭐 똑같긴 한데 이 방법도 있습니다.. ㅋㅋㅋ

---

저도 지금 처음 해보는 거라 실수가 많습니다ㅜ `No such file or directory` 에러가 뜨는 걸 보니 directory가 있어야 할 것 같습니다. Sion님이 제안해주신 방법으로 한번 해보겠습니다!